### PR TITLE
Add additional doc for CSI command:

### PR DESCRIPTION
**Pull request by jichenjc**
list-volumes
create-snapshot
list-snapshots
delete-snapshot

are covered by using csc tools.

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: docs/using-cinder-csi-plugin.md [hunk 1]
```diff
@@ -256,13 +256,61 @@ $ csc controller create-volume --endpoint tcp://127.0.0.1:10000 CSIVolumeName
 "8a55f98f-e987-43ab-a9f5-973352bee19c"  1073741824      "availability"="nova"
 ```
 
+#### List volumes
+Following smaple list all volumes:
```
Comment (ramineni) [inline docs/using-cinder-csi-plugin.md]: :nit: s/smaple/sample

File: docs/using-cinder-csi-plugin.md [hunk 2]
```diff
@@ -256,13 +256,61 @@ $ csc controller create-volume --endpoint tcp://127.0.0.1:10000 CSIVolumeName
...
+
 #### Delete a volume
 Following sample deletes a volume ``01217e93-bd1b-4760-b5d8-18b8b3d47f91``
 ```
 $ csc controller delete-volume --endpoint tcp://127.0.0.1:10000 01217e93-bd1b-4760-b5d8-18b8b3d47f91
 01217e93-bd1b-4760-b5d8-18b8b3d47f91
 ```
 
+#### Create a snapshot from volume
+Following sample create a snapshot from volume `40615da4-3fda-4e78-bf58-820692536e68`.
```
Comment (NickrenREN) [inline docs/using-cinder-csi-plugin.md]: s/create/creates

File: docs/using-cinder-csi-plugin.md [hunk 3]
```diff
@@ -256,13 +256,61 @@ $ csc controller create-volume --endpoint tcp://127.0.0.1:10000 CSIVolumeName
...
++--------------------------------------+------+-------------+-----------+------+
+| ID                                   | Name | Description | Status    | Size |
++--------------------------------------+------+-------------+-----------+------+
+| e2df8c2a-58eb-40fb-8ec9-45aee5b8f39f | s1   | None        | available |    1 |
++--------------------------------------+------+-------------+-----------+------+
+```
+
+#### List snapshots
+
+Following sample list all snapshots:
```
Comment (NickrenREN) [inline docs/using-cinder-csi-plugin.md]: ditto

File: docs/using-cinder-csi-plugin.md [hunk 4]
```diff
@@ -256,13 +256,61 @@ $ csc controller create-volume --endpoint tcp://127.0.0.1:10000 CSIVolumeName
...
+```
+$ csc controller  list-snapshots --endpoint tcp://127.0.0.1:10000
+"e2df8c2a-58eb-40fb-8ec9-45aee5b8f39f" 1073741824      40615da4-3fda-4e78-bf58-820692536e68    seconds:1561532425      true
+```
+
+#### Delete a snapshot
+
+Following sample delete the snapshot `e2df8c2a-58eb-40fb-8ec9-45aee5b8f39f`.
```
Comment (NickrenREN) [inline docs/using-cinder-csi-plugin.md]: ditto

File: docs/using-cinder-csi-plugin.md [hunk 5]
```diff
@@ -256,13 +256,61 @@ $ csc controller create-volume --endpoint tcp://127.0.0.1:10000 CSIVolumeName
...
 
+#### Create a snapshot from volume
+Following sample create a snapshot from volume `40615da4-3fda-4e78-bf58-820692536e68`.
+After execution, snapshot `e2df8c2a-58eb-40fb-8ec9-45aee5b8f39f` will be created.
+```
+$ csc controller create-snapshot --source-volume 40615da4-3fda-4e78-bf58-820692536e68 --endpoint tcp://127.0.0.1:10000 s1
+"e2df8c2a-58eb-40fb-8ec9-45aee5b8f39f"  1073741824      40615da4-3fda-4e78-bf58-820692536e68    seconds:1561530261      true
+```
+
+use openstack command to verify:
```
Comment (NickrenREN) [inline docs/using-cinder-csi-plugin.md]: uppercase ?

File: docs/using-cinder-csi-plugin.md (hunk 1 above)
Comment (jichenjc) [reply docs/using-cinder-csi-plugin.md]: done

File: docs/using-cinder-csi-plugin.md (hunk 2 above)
Comment (jichenjc) [reply docs/using-cinder-csi-plugin.md]: done

File: docs/using-cinder-csi-plugin.md (hunk 5 above)
Comment (jichenjc) [reply docs/using-cinder-csi-plugin.md]: done

**Pull request merged**